src/__tests__/__snapshots__/App.js.snap

```diff
@@ -759,28 +759,7 @@ exports[`App should render dashboard 1`] = `
         <div
           data-testid="openapi-layout"
           id="swaggerContainer"
-        >
-          <div
-            class="swagger-ui"
-            data-reactroot=""
-          >
-            <div
-              class="loading-container"
-            >
-              <div
-                class="info"
-              >
-                <div
-                  class="loading-container"
-                >
-                  <div
-                    class="loading"
-                  />
-                </div>
-              </div>
-            </div>
-          </div>
-        </div>
+        />
```

Comment:
Do you know why this changed? It seems fine though. Maybe the test is somehow waiting for the component to finish loading now.

Reply:
It is because I started to mock swagger-ui, so the loading component stopped being in the snapshot